### PR TITLE
Added MVP draft.

### PR DESCRIPTION
**Pull request by skullbunnygalaxy**
Looking for feedback on the MVP draft.

### CODE REVIEW

File: roadmap/mvp.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+6. Commission history page for denizens. Same as above, pretty much, for now.
+7. Payment Processing integration (ie stripe mentioned above)
+
+## MVP social features:
+- ability to follow a user or a studio
+- paginated streams (start with default stream for anyone you are following) that shows updates from anyone you are following (ie user bought commission from studio, studio posted this artwork)
+- way for studio to post update or new art piece to stream for their followers (in case of art piece default posts to their gallery)
+
+## Minor needed features
+- way to limit sign ups starting out
```
Comment (mattvanhorn) [inline roadmap/mvp.md:27]: Invitation only, perhaps?

File: roadmap/mvp.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- mailing list for those wanting to sign up
+- email verification for new accounts
+
+
+## Waiting till after MVP
+- showcases
+- ability to create custom streams
```
Comment (mattvanhorn) [inline roadmap/mvp.md:34]: Could you explain these first two just a bit more?

**Review by mattvanhorn — APPROVED** [review]

File: roadmap/mvp.md (hunk 1 above)
Comment (skullbunnygalaxy) [reply roadmap/mvp.md:27]: Yes, invitation-only is the main way we've been discussing doing this. If we're going that route, then we'll need to set up a way to send + track invites and allow users to use them to create an account.

File: roadmap/mvp.md (hunk 2 above)
Comment (skullbunnygalaxy) [reply roadmap/mvp.md:34]: Showcases are curated collections of art. The idea behind this is similar to [Steam Curators](https://store.steampowered.com/curators/topcurators/). Users can highlight recommended pieces based on a topic or theme, which they can share with other users in order to help users find artists they may want to follow. These can also be used by artists to show a collection of pieces related to a game of artist telephone, which is a game where artists take turns drawing a character/scenario that they only know the description of as given to them by the previous artist. 

Steams are our version of the live feed you see on most sites. I'm named it streams as a reference to the [digital garden movement](https://maggieappleton.com/garden-history), which often refers to feeds as streams. However, I am hoping to design ours to be more user-friendly than the typical feed. 
One of the features that I regularly hear people describe missing from the old web a la LiveJournal is the ability to create custom feeds front and center, including adjusting their default feed. I would like users to be able to create multiple streams based on what kind of content they are in the mood to view. They could have separate lists for cozy art, political art, fluffy art, specific fan art pairings, etc. The lists would let them add specific artists + topic tags. 
For instance, you know you like artist A, but they have multiple topics that you are not in the mood to see all the time. You can set up the stream to show artwork from artist A within specific topics. You can also subscribe to that overall topic for that stream. If there is someone posting within that topic that you do not want to see, you can set an exclude to remove those pieces. Alternatively, you can set it up to where you only see pieces tagged with that topic for artists you follow.

**Pull request merged**